portfolio/src/main/webapp/script.js

```diff
@@ -95,6 +99,39 @@ function displayImage(imageID) {
     pic.style.display = "block";
 }
 
+/*
+ * Handles branching animation off of the timeline.
+ */
+function displayTimelineBranch(branchName, positionToExtend) {
+    var branch = document.getElementById(branchName);
+    var pos = 50;
+
+    var id = setInterval(extendTimelineBranch, 20);
```

Comment:
IMO, "id" does not describe it's purpose properly. Please use a more descriptive name.